picobot/handlers.py

```diff
@@ -0,0 +1,289 @@
+import logging
+import dataset
+from telegram import Bot, Update, Message
+
+from functools import wraps
+
+from .config import CREATOR_ID, DB_PATH, ROOT_DIR
+from picobot import responses
+from .msg_type import MsgType
+from .painter import sticker_from_text, sticker_from_image
+
+IMG_DIR = ROOT_DIR + '/images/'
+IMG_NAME = 'img'
+AVATAR_NAME = 'avatar'
+
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO
```

Comment:
Be consistent on quotes: use only single-quotes (preffered) or only double-quotes.